backend/app/controllers/projects.js

```diff
@@ -373,6 +415,51 @@ module.exports = {
                     .update(updateObj)
                     .where(Project.id.equals(req.params.id))
                 );
+
+                if (result) { // If the status was changed
+                    if (updateObj.status) { // it was made active
```

Comment:
I haven't been receiving emails for inactive, and I think this is why. Within this scope you're checking for the status again to decide which email to send, and this higher condition only sees the actives.

Reply:
Good catch! Fixed!